valhalla/jawn/src/lib/handlers/HandlerContext.ts

```diff
@@ -163,13 +165,17 @@ export const toHeliconeRequest = (context: HandlerContext): HeliconeRequest => {
     provider: context.message.log.request.provider,
     delay_ms: context.message.log.response.delayMs ?? null,
     time_to_first_token: context.message.log.response.timeToFirstToken ?? null,
-    total_tokens: context.usage.totalTokens ?? null,
-    prompt_tokens: context.usage.promptTokens ?? null,
-    completion_tokens: context.usage.completionTokens ?? null,
-    prompt_cache_write_tokens: context.usage.promptCacheWriteTokens ?? null,
-    prompt_cache_read_tokens: context.usage.promptCacheReadTokens ?? null,
-    prompt_audio_tokens: context.usage.promptAudioTokens ?? null,
-    completion_audio_tokens: context.usage.completionAudioTokens ?? null,
+    
+    // We don't track tokens on cache hits, since cost is 0
+    total_tokens: context.message.log.request.cacheReferenceId != DEFAULT_UUID ? 0 : context.usage.totalTokens ?? null,
```

Comment:
nit pull `context.message.log.request.cacheReferenceId != DEFAULT_UUID` to a variable like `isCachedRequest` so reader knows why we are doing this